src/main/java/teammates/logic/core/GoogleCloudLoggingService.java

```diff
@@ -241,13 +258,41 @@ private void createLogEntry(LogEntry entry) throws LogServiceException {
         if (s.endTime != null) {
             logFilters.add("timestamp<=\"" + s.endTime.toString() + "\"");
         }
-        if (s.severities != null) {
-            String severitiesFilter = s.severities.stream().collect(Collectors.joining(" OR "));
-            logFilters.add("severity=(" + severitiesFilter + ")");
+        if (s.severity != null) {
+            logFilters.add("severity=" + s.severity);
         }
-        if (s.minSeverity != null && s.severities == null) {
+        if (s.minSeverity != null && s.severity == null) {
             logFilters.add("severity>=" + s.minSeverity.toString());
         }
+        if (s.traceId != null) {
+            logFilters.add("trace=\"" + s.traceId + "\"");
+        }
+        if (s.apiEndpoint != null) {
+            logFilters.add("jsonPayload.actionClass=\"" + s.apiEndpoint + "\"");
```

Comment:
Why name the parameter `apiEndpoint` when the thing being passed is the action class?

Reply:
It is updated now.